src/components/repl.tsx

```diff
@@ -192,46 +229,53 @@ const Repl: ReplProps = (props) => {
         <TabList>
           <For each={props.tabs}>
             {(tab, index) => (
-              <TabItem active={props.current === tab.name} class="mr-2">
-                <div
-                  ref={(el) => tabRefs.set(tab.name, el)}
-                  class="cursor-pointer select-none rounded border border-solid border-transparent py-2 px-3 transition focus:border-blue-600 focus:outline-none"
-                  contentEditable={edit() == index()}
-                  onBlur={(e) => {
-                    setEdit(-1);
-                    setCurrentName(e.currentTarget.textContent!);
-                  }}
-                  onKeyDown={(e) => {
-                    if (e.code === 'Space') e.preventDefault();
-                    if (e.code !== 'Enter') return;
-                    setEdit(-1);
-                    setCurrentName(e.currentTarget.textContent!);
-                  }}
-                  onClick={() => setCurrentTab(tab.name)}
-                  onDblClick={(e) => {
-                    e.preventDefault();
-                    setEdit(index());
-                    tabRefs.get(tab.name)?.focus();
-                  }}
-                >
-                  {tab.name}
-                </div>
-
-                <Show when={index() > 0}>
-                  <button
-                    type="button"
-                    class="cursor-pointer"
-                    onClick={() => {
-                      removeTab(tab.name);
+              <Show when={!tab.name.startsWith('data_')}>
```

Comment:
I believe this is more cleanly represented by a .filter in the For, although I'm not fully confident in that

Reply:
I've just implemented this too